From 40516e49e13658f99b30ad6bff3d9239a210fbe7 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Thu, 14 Dec 2017 23:00:56 +0900
Subject: [PATCH 132/909] clk: renesas: r8a7795: Add iVDP1C clocks to ES1.x

commit 24f415e5b323ee570cbac2f4aca5496cd393cbe0 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds iVDP1C and STB clocks to the R8A7795 ES1.x SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/renesas/r8a7795-cpg-mssr.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/clk/renesas/r8a7795-cpg-mssr.c b/drivers/clk/renesas/r8a7795-cpg-mssr.c
index 6af729c46d80..a386bfe65baa 100644
--- a/drivers/clk/renesas/r8a7795-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a7795-cpg-mssr.c
@@ -120,10 +120,12 @@ static struct cpg_core_clk r8a7795_core_clks[] __initdata = {
 };
 
 static struct mssr_mod_clk r8a7795_mod_clks[] __initdata = {
+	DEF_MOD("stb",			 104,	R8A7795_CLK_S2D1), /* ES1.x */
 	DEF_MOD("3dge",			 112,	R8A7795_CLK_ZG),
 	DEF_MOD("fdp1-2",		 117,	R8A7795_CLK_S2D1), /* ES1.x */
 	DEF_MOD("fdp1-1",		 118,	R8A7795_CLK_S0D1),
 	DEF_MOD("fdp1-0",		 119,	R8A7795_CLK_S0D1),
+	DEF_MOD("ivdp1c",		 128,	R8A7795_CLK_S2D1), /* ES1.x */
 	DEF_MOD("vcpl4",		 129,	R8A7795_CLK_S2D1), /* ES1.x */
 	DEF_MOD("vcplf",		 130,	R8A7795_CLK_S2D1),
 	DEF_MOD("vdpb",			 131,	R8A7795_CLK_S2D1),
@@ -403,7 +405,9 @@ static const struct mssr_mod_reparent r8a7795es1_mod_reparent[] __initconst = {
 	 */
 
 static const unsigned int r8a7795es2_mod_nullify[] __initconst = {
+	MOD_CLK_ID(104),			/* STB */
 	MOD_CLK_ID(117),			/* FDP1-2 */
+	MOD_CLK_ID(128),			/* iVDP1C */
 	MOD_CLK_ID(129),			/* VCPL4 */
 	MOD_CLK_ID(327),			/* USB3-IF1 */
 	MOD_CLK_ID(600),			/* FCPVD3 */
-- 
2.17.1

